### DISCUSSION TITLE
Deploy to Netlify documentation is a bit confusing

### DISCUSSION THREAD

**Discussion by HeyMarcy** [ctrl-alt-del-world/midway#38] [Q&A]
Thanks so much for this repo!
I'm able to get up and running locally, but I'm having trouble deploying to Netlify.  The docs under 'Deploy to Netlify' state: 
> Build command: npm run build
> Publish directory: web/public
> You will need to set the publish directory in Netlify to web.

The last two sentences contradict each other.

I set the "Base" directory to `web`
here's the deploy log:

12:33:27 PM:   1. build.command from netlify.toml                            
12:33:27 PM: ─────────────────────────────────────────────────────────
12:33:27 PM: ​
12:33:27 PM: $ gatsby build && netlify-lambda build src/lambda
12:33:30 PM: success open and validate gatsby-configs - 0.096s
12:33:31 PM: error Error in "/opt/build/repo/web/gatsby-node.js": Configuration must contain `projectId`
12:33:31 PM: 
12:33:31 PM: 
12:33:31 PM:   Error: Configuration must contain `projectId`
12:33:31 PM:   
12:33:31 PM:   - config.js:51 exports.initConfig
12:33:31 PM:     [web]/[@sanity]/client/lib/config.js:51:11
12:33:31 PM:   
12:33:31 PM:   - sanityClient.js:78 SanityClient.config
12:33:31 PM:     [web]/[@sanity]/client/lib/sanityClient.js:78:25
12:33:31 PM:   
12:33:31 PM:   - sanityClient.js:46 new SanityClient
12:33:31 PM:     [web]/[@sanity]/client/lib/sanityClient.js:46:8
12:33:31 PM:   
12:33:31 PM:   - sanityClient.js:43 SanityClient
12:33:31 PM:     [web]/[@sanity]/client/lib/sanityClient.js:43:12
12:33:31 PM:   
12:33:31 PM:   - sanity.js:3 Object.<anonymous>
12:33:31 PM:     /opt/build/repo/web/src/api/sanity.js:3:18
12:33:31 PM:   
12:33:31 PM:   - v8-compile-cache.js:192 Module._compile
12:33:31 PM:     [web]/[v8-compile-cache]/v8-compile-cache.js:192:30
12:33:31 PM:   
12:33:31 PM:   - loader.js:1158 Object.Module._extensions..js
12:33:31 PM:     internal/modules/cjs/loader.js:1158:10
12:33:31 PM:   
12:33:31 PM:   - loader.js:986 Module.load
12:33:31 PM:     internal/modules/cjs/loader.js:986:32
12:33:31 PM:   
12:33:31 PM:   - loader.js:879 Function.Module._load
12:33:31 PM:     internal/modules/cjs/loader.js:879:14
12:33:31 PM:   
12:33:31 PM:   - loader.js:1026 Module.require
12:33:31 PM:     internal/modules/cjs/loader.js:1026:19
12:33:31 PM:   
12:33:31 PM:   - v8-compile-cache.js:159 require
12:33:31 PM:     [web]/[v8-compile-cache]/v8-compile-cache.js:159:20
12:33:31 PM:   
12:33:31 PM:   - gatsby-node.js:5 Object.<anonymous>
12:33:31 PM:     /opt/build/repo/web/gatsby-node.js:5:16
12:33:31 PM:   
12:33:31 PM:   - v8-compile-cache.js:192 Module._compile
12:33:31 PM:     [web]/[v8-compile-cache]/v8-compile-cache.js:192:30
12:33:31 PM:   
12:33:31 PM:   - loader.js:1158 Object.Module._extensions..js
12:33:31 PM:     internal/modules/cjs/loader.js:1158:10
12:33:31 PM:   
12:33:31 PM:   - loader.js:986 Module.load
12:33:31 PM:     internal/modules/cjs/loader.js:986:32
12:33:31 PM:   
12:33:31 PM:   - loader.js:879 Function.Module._load
12:33:31 PM:     internal/modules/cjs/loader.js:879:14
12:33:31 PM:   
12:33:31 PM: 
12:33:31 PM: not finished load plugins - 0.946s
12:33:31 PM: ​
12:33:31 PM: ────────────────────────────────────────────────────────────────
12:33:31 PM:   "build.command" failed                                        
12:33:31 PM: ────────────────────────────────────────────────────────────────
12:33:31 PM: ​
12:33:31 PM:   Error message
12:33:31 PM:   Command failed with exit code 1: gatsby build && netlify-lambda build src/lambda
12:33:31 PM: ​
12:33:31 PM:   Error location
12:33:31 PM:   In build.command from netlify.toml:
12:33:31 PM:   gatsby build && netlify-lambda build src/lambda
12:33:31 PM: ​
12:33:31 PM:   Resolved config
12:33:31 PM:   build:
12:33:31 PM:     base: /opt/build/repo/web
12:33:31 PM:     command: gatsby build && netlify-lambda build src/lambda
12:33:31 PM:     commandOrigin: config
12:33:31 PM:     environment:
12:33:31 PM:       - TYPE
12:33:31 PM:     publish: /opt/build/repo/web/public
12:33:31 PM:   functions:
12:33:31 PM:     '*': {}
12:33:31 PM:   functionsDirectory: /opt/build/repo/web/functions`

**Comment by Hahlh** [comment]
Hey there :)

This seems like it is mainly an issue with missing config:
``` 12:33:31 PM: error Error in "/opt/build/repo/web/gatsby-node.js": Configuration must contain projectId```
